apps/redi-connect/src/components/AppNotification.tsx

```diff
@@ -99,7 +99,7 @@ export function AppNotification() {
 
   const styleClasses = useNotificationStyles()
 
-  const Icon = state ? variantIcon[state.variant] : null
+  const Icon = variantIcon[state?.variant] || null
```

Comment:
The one on top misses the case of not having a `variantIcon` with that `variant` name, returning `undefined` instead of `null`, and leading to 3 possible return types. 

A junior benefits from being in contact with code that is not made for dummies and that enforces good practices. That's what my experience as Tech Lead and Senior has been taught me. on top you have three elements to analyze; down just two, and one is a fallback, so I can focus on just one element.